torch/nn/utils/rnn.py

```diff
@@ -130,10 +150,12 @@ def pad_packed_sequence(sequence, batch_first=False, padding_value=0):
         padding_value (float, optional): values for padded elements.
 
     Returns:
-        Tuple of Variable containing the padded sequence, and a list of lengths
-        of each sequence in the batch.
+        Tuple of Variable containing the padded sequence, and Variable
+        containing the list of lengths of each sequence in the batch.
```

Comment:
I don't think we want to make this BC-breaking change. CC @apaszke what do you think?

Reply:
I’d rather avoid it unless it’s 100% necessary. Is packed sequence support so important? Does ONNX even support variable length sequences?

Reply:
ONNX does support variable length sequences (though it's done differently, with a separate, optional `sequence_lens` input).

I believe it is 100% necessary if we want to support variable-length sequences. pytorch itself has some invariants that model inputs/outputs must be (lists of, tuples of) Variables, and PackedSequence violates that. None of the jit/tracing/symbolic machinery works with the current representation.

I am open to investigating alternatives, if anyone has suggestions/ideas of how else this could be made to work.